src/3rd_party_repos.c

```diff
@@ -183,15 +183,15 @@ static int overwrite_repo_config_file(struct list *repos)
 	// open it in trunc mode as we re-write all the contents again
 	fp = fopen(repo_config_file_path, "w");
 	if (!fp) {
-		error("Impossible to write to file %s", repo_config_file_path);
+		error("Impossible to write to file %s\n", repo_config_file_path);
```

Comment:
Unable to write to the file?

Reply:
Well, my commit didn't really modified that message I only added a missing line break, I'm not too concerned with this error message cause I don't expected it to be shown a lot. Having said that, sure, I could change it to what you suggests, the current error message seems a little odd.